<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerAddDialog</class>
 <widget class="QDialog" name="ServerAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new server</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>label</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Host:</string>
       </property>
       <property name="buddy">
        <cstring>hostEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="hostEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Port:</string>
       </property>
       <property name="buddy">
        <cstring>portEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="portEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Custom &amp;user:</string>
       </property>
       <property name="buddy">
        <cstring>userEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="userEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="protocolLabel">
       <property name="text">
        <string>Protocol:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="protocolPlain">
       <property name="text">
        <string>TCP/IP</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QRadioButton" name="protocolSsl">
       <property name="text">
        <string>TCP/IP with SSL</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QRadioButton" name="protocolHttp">
       <property name="text">
        <string>HTTP</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QRadioButton" name="protocolHttps">
       <property name="text">
        <string>HTTPS</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="MessageLabel" name="sslMessageLabel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="addToCurrentCb">
     <property name="text">
      <string>Add server to current &amp;view</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QWidget</extends>
   <header>MessageLabel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServerAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServerAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
